roles/image_builder/README.md

```diff
@@ -11,6 +11,55 @@ None
 Role Variables
 --------------
 
+## microshift_rhsm_repos
+
+Type: list
+Required: false
+
+*Use microshift_rhsm_repos to set the rhsm repos used for the Microshift package*
+
+List of RHSM repositories to make available to the 
```

Comment:
Let's make this mention of RHSM a link to here for the passerby who might not know what RHSM is.

https://access.redhat.com/products/red-hat-subscription-management/

so ... `[RHSM|https://access.redhat.com/products/red-hat-subscription-management/]` instead of just `RHSM` in the markdown (for clarity in case I described that badly).